code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -527,3 +527,45 @@
 /obj/effect/mob_spawn/human/oldsci/Destroy()
 	new/obj/structure/showcase/machinery/oldpod/used(drop_location())
 	return ..()
+
+
+#define PIRATE_NAMES_FILE "pirates.json"
+
+/obj/effect/mob_spawn/human/pirate
+	name = "space pirate sleeper"
+	desc = "A cryo sleeper smelling faintly of rum."
+	random = TRUE
+	icon = 'icons/obj/cryogenic2.dmi'
+	icon_state = "sleeper"
+	mob_name = "a space pirate"
+	mob_species = /datum/species/human
+	outfit = /datum/outfit/pirate
+	roundstart = FALSE
+	death = FALSE
+	anchored = TRUE
+	density = FALSE
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are an space pirate. The station refused to pay for your protection, protect the ship,siphon the credits from the station and raid it for even more loot.</b>"
```

Comment:
# YOU ARE A PIRATE
## YARR HARR 
### FIDDLE DEE DEE
**DO WHAT YOU WANT COS A PIRATE IS FREE**

no but really, "you are **_AN_** pirate" should be "a"

Reply:
Tbh, i copypasted the start from ashwalkers. Most of flavour strings are pretty shit atm. Will fix.